src/app/plugins/tictactoe.plugin.ts

```diff
@@ -92,24 +96,27 @@ class TTTGame {
 
   private _playerA: User;
   private _playerB: User;
+  private _playingLion: boolean;
   private _board: number[][];
   private _flagToEmoji: Record<number, string> = {
     [-1]: ':regional_indicator_o:',
     [0]: '🟦',
     [1]: ':regional_indicator_x:',
   };
 
-  private _choosing: Choosing = Choosing.Row;
+  private _choosing: Choosing = Choosing.Column;
   private _winner: Maybe<number> = null;
+  private _gameOver: boolean = false;
   private _row = -1;
   private _col = -1;
 
   // -1 is playerA
   private currentPlayer: number = -1;
 
-  constructor(playerA: User, playerB: User) {
+  constructor(playerA: User, playerB: User, playingLion: Maybe<boolean>) {
     this._playerA = playerA;
     this._playerB = playerB;
+    this._playingLion = playingLion ? true : false;
```

Comment:
Why is playing Lion `Maybe<...>`
Then you could just do `this._playingLion = playingLion`

Reply:
PlayingLion is maybe because of the null ability of the members cache. Depending on that helper function mentioned above though we might be able to fix this.

Reply:
Resolved with the change mentioned above